huaweicloud/resource_huaweicloud_identity_acl_test.go

```diff
@@ -0,0 +1,245 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/huaweicloud/golangsdk/openstack/identity/v3.0/acl"
+)
+
+func TestAccIdentitACL_basic(t *testing.T) {
+	var acl acl.ACLPolicy
+	resourceName := "huaweicloud_identity_acl.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPrechecRegion(t)
```

Comment:
as iam should have the admin privileges, please use `testAccPreCheckAdminOnly`.

Reply:
Added `testAccPreCheckAdminOnly`.